src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,7 +5,14 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String model;
```

Comment:
The field 'model' should have an access modifier to encapsulate the field properly. Consider using private and providing a getter (and possibly a setter) for it.